src/main/java/seedu/address/logic/commands/patient/ListPatientCommand.java

```diff
@@ -12,10 +12,9 @@
  */
 public class ListPatientCommand extends Command {
 
-    public static final String COMMAND_WORD = "list";
-
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String COMMAND_WORD = "listpatient";
 
+    public static final String MESSAGE_SUCCESS = "Listed all patients.";
```

Comment:
This sounds a bit weird to me. Maybe change it to "All patients are listed" or " list all patients"

Reply:
Sure, I will change to "All patients are listed."